chef/data_bags/crowbar/migrate/nova/011_configure_block_device_allocation.rb

```diff
@@ -0,0 +1,11 @@
+def upgrade ta, td, a, d
+  a['block_device_allocation_retries'] = 60
+  a['block_device_allocation_retries_interval'] = 3
+  return a, d
+end
+
+def downgrade ta, td, a, d
+  a.delete 'block_device_allocation_retries'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.